applications/minotari_console_wallet/src/automation/commands.rs

```diff
@@ -463,6 +468,7 @@ pub async fn make_it_rain(
                                 UtxoSelectionCriteria::default(),
                                 address.clone(),
                                 msg.clone(),
+                                PaymentId::Zero,
```

Comment:
```suggestion
                                PaymentId::Empty,
```
I prefer "empty" because zero makes it sound like it's a number